docs/03-tutorials/00-api-exposure/apix-03-expose-multiple-services.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Expose multiple services on the same host
+---
+
+This tutorial shows how to expose a workload with multiple services that share the same host on different paths.
```

Comment:
it's not quite clear to me what `a workload with multiple services` means. i would just go simply with multiple services
```suggestion
This tutorial shows how to expose multiple services that share the same host on different paths.
```